templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="nav justify-content-center">
+  <li class="nav-item">
+    <a class="nav-link active" aria-current="page" href="#">Home page</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link active" aria-current="page" href="#">Manufacturers</a>
```

Comment:
The `href` attribute is set to `#`, which is a placeholder. Consider updating it to point to the actual URL for the Manufacturers page.